prow/config.yaml

```diff
@@ -2931,7 +2931,7 @@ periodics:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
       - name: USER
-        value: prow
+        value: jenkins
```

Comment:
https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L180-L188

that is only a Jenkins trigger, the job is actually running on Jenkins, thus will have USER=jenkins set from the Jenkins VMs.